client/reader/stream/helper.js

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { some, startsWith } from 'lodash';
```

Comment:
Could we use [`Array.prototype.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) and [`String.prototype.startsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) instead? We've been trying to reduce and remove Lodash for a while now.